src/services/SendEmailService.ts

```diff
@@ -70,72 +71,94 @@ export class SendEmailService {
 	 * @returns EmailResponse
 	 * @throws AppError
 	 */
-	async sendEmail(message: Email): Promise<EmailResponse> {
+	async sendYotiPdfEmail(message: Email): Promise<EmailResponse> {
 		// Fetch the instructions pdf from Yoti
 		try {
 			const encoded = await this.fetchInstructionsPdf(message);
 			if (encoded) {
 				this.logger.debug("sendEmail", SendEmailService.name);
-				return await this.sendGovNotification(message, encoded);
+				this.logger.info("Sending Yoti PDF email");
+
+				const options = {
+					personalisation: {
+						"first name": message.firstName,
+						"last name": message.lastName,
+						"link_to_file": { "file": encoded, "confirm_email_before_download": true, "retention_period": "2 weeks" },
+					},
+					reference: message.referenceId,
+				};
+
+				const emailResponse = await this.sendGovNotification(this.environmentVariables.getPdfEmailTemplateId(this.logger), message, options);
+				await this.sendF2FYotiEmailedEvent(message);
+				return emailResponse;
 			} else {
-				this.logger.error("sendEmail - Failed to fetch the Instructions pdf");
-				throw new AppError(HttpCodesEnum.SERVER_ERROR, "sendEmail - Failed to fetch the Instructions pdf");
+				this.logger.error("sendYotiPdfEmail - Failed to fetch the Instructions pdf");
```

Comment:
Do we need a messageCode for this error?

Reply:
Done